salt/modules/minion.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide information about minions
+'''
+
+# Import Python libs
+import os
+
+# Import Salt libs
+import salt.utils
+import salt.key
+
+
+def list():
+    '''
+    Return a list of accepted, denied, unaccepted and rejected keys.
+    This is the same output as `salt-key -L`
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'master' minion.list
+    '''
+    pki_dir = globals().get('__salt__')['config.get']('pki_dir', '')
```

Comment:
The loader automatically injects `__salt__` into the module's namespace. Instead, this should be simply `__salt__['config.get']('pki_dir', '')`